src/libraries/System.Net.Sockets/src/System/Net/Sockets/SocketAsyncEngine.Unix.cs

```diff
@@ -293,13 +175,11 @@ private void EventLoop()
         {
             try
             {
-                bool shutdown = false;
                 Interop.Sys.SocketEvent* buffer = _buffer;
                 ConcurrentDictionary<IntPtr, SocketAsyncContextWrapper> handleToContextMap = _handleToContextMap;
                 ConcurrentQueue<SocketIOEvent> eventQueue = _eventQueue;
-                IntPtr shutdownHandle = ShutdownHandle;
                 SocketAsyncContext? context = null;
-                while (!shutdown)
+                while (true)
```

Comment:
This will conflict with #37138 but I suppose the merge order doesn't make any difference.

Reply:
merge conficts are a fact of developer life :)